server/src/services/app_import_export.service.ts

```diff
@@ -450,7 +463,12 @@ export class AppImportExportService {
                     const newLayout = new Layout();
                     newLayout.type = type;
                     newLayout.top = layout.top;
-                    newLayout.left = layout.left;
+                    newLayout.left =
+                      layout.dimensionUnit !== 'count'
```

Comment:
Create enum for `dimensionUnit`

Reply:
Can create an enum like this
https://github.com/ToolJet/tj-ee/blob/develop/server/src/helpers/user_lifecycle.ts#L3

Reply:
Best if we can move component related utils to different file, other than utils.helper.ts
can create a directory - `components` under /src/helper/ and add types and utils files there